frontend/src/components/Application/Application.tsx

```diff
@@ -0,0 +1,362 @@
+import React, { useState } from "react";
+import { Accordion, Documents, Form, Header, SmallButton } from "../UI";
+import img1 from "../../assets/images/Id.jpeg";
+import img2 from "../../assets/images/Payslip.jpg";
+import img3 from "../../assets/images/Meldezettel.png";
+
+const mockData = [
```

Comment:
Would it be a cleaner approach to load the mock data from a resource file rather than putting it in the actual code?

Reply:
I agree. Moved mock data to file